### DISCUSSION TITLE
Facing invalid path 'Machine_Learning/Hand Game Controller/"A" Key Binding.png' while sync fork and pull origin

### DISCUSSION THREAD

**Discussion by Mayank202004** [UTSAVS26/PyVerse#115] [Q&A] [answered]
Facing invalid path 'Machine_Learning/Hand Game Controller/"A" Key Binding.png' while sync fork and pull origin.
Tried re cloning the repo.

**Comment by Mayank202004** [comment]
Maybe issue due to double inverted commas in file name  "A" Key Binding.png in project Machine_Learning/Hand Game Controller

**Comment by UTSAVS26** [comment, marked as answer]
Yes that's the main reason why many contributors are facing this issue, I have already revert back the changes and asked @aviralgarg05 to update it as soon as possible.

**Reply by Mayank202004** [reply]
Thanks
